Views/Metas.liquid

```diff
@@ -0,0 +1,31 @@
+﻿{% Style "~/Themes/TryOrchard.Theme/Styles/fonts.css" %}
+{% assign faviconUrl = "/Themes/TryOrchard.Theme/Images/Favicons/" %}
+
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="shortcut icon" type="image/x-icon" href="{{ faviconUrl }}favicon.ico" />
+<link rel="image/png" sizes="36x36" href="{{ faviconUrl }}android-chrome-36x36.png" />
```

Comment:
Ezek az URL-ek nem fognak localban (local fájlokkal a témában), stagingben és prodban (remote fájlokkal blob storage-ben) egyaránt működni. Ahhoz a SetLink, SetMeta tageket kell használni, ahogy amúgy is csinálnád, lásd a Readme-t: https://github.com/Lombiq/Orchard-Liquid-Markup Alternatíva a Href tag használata, talán ez az egyszerűbb, így:

```
{% capture faviconBaseUrl %}{% Href "~/Themes/MyTheme/Images/Favicons/" %}{% endcapture %}

<link rel="shortcut icon" type="image/x-icon" href="{{ faviconBaseUrl }}favicon.ico" />
...
```